### DISCUSSION TITLE
请问朱江大佬为啥要放弃使用和风天气的sdk啊？

### DISCUSSION THREAD

**Discussion by qwertystar** [zhujiang521/PlayWeather#13] [Q&A] [answered]
我在看git的提交记录里面看到说作者已经放弃使用和风天气的专门的sdk了，请教一下为啥啊？

**Comment by zhujiang521** [comment, marked as answer]
sdk 中有的接口调用有问题，给官方反应问题，官方说他们的接口没问题，说我的调用不对。。。但那么多接口都可以，就几个不行，而且是高版本安卓中才会有问题，他们不改。。然后还有现成的api ，然后就不用他们的sdk 而自己写了下发自我的 iPhone在 2023年10月17日，08:25，qwertystar ***@***.***> 写道：﻿
我在看git的提交记录里面看到说作者已经放弃使用和风天气的专门的sdk了，请教一下为啥啊？
